packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/overview/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,36 @@
+{
+  "octavia_load_balancer_pools_detail_overview_management_title": "Gestion",
+  "octavia_load_balancer_pools_detail_overview_management_edit": "Editer",
+  "octavia_load_balancer_pools_detail_overview_management_add_manually": "Ajouter un membre manuellement",
+  "octavia_load_balancer_pools_detail_overview_management_add_ip_instance": "Ajouter une IP d'instance",
+  "octavia_load_balancer_pools_detail_overview_management_delete": "Supprimer",
+  "octavia_load_balancer_pools_detail_overview_info_title": "Informations",
+  "octavia_load_balancer_pools_detail_overview_info_name": "Nom",
+  "octavia_load_balancer_pools_detail_overview_info_pool_id": "ID du Pool",
+  "octavia_load_balancer_pools_detail_overview_info_listener_id": "ID du listener associé",
+  "octavia_load_balancer_pools_detail_overview_info_provisioning_status": "Statut d'approvisionnement",
+  "octavia_load_balancer_pools_detail_overview_info_provisioning_status_active": "actif",
+  "octavia_load_balancer_pools_detail_overview_info_provisioning_status_creating": "en cours de création",
+  "octavia_load_balancer_pools_detail_overview_info_provisioning_status_deleted": "supprimé",
+  "octavia_load_balancer_pools_detail_overview_info_provisioning_status_deleting": "en cours de suppression",
+  "octavia_load_balancer_pools_detail_overview_info_provisioning_status_error": "erreur",
+  "octavia_load_balancer_pools_detail_overview_info_provisioning_status_updating": "en cours de mise à jour",
+  "octavia_load_balancer_pools_detail_overview_info_operating_status": "Statut d'opération",
+  "octavia_load_balancer_pools_detail_overview_info_operating_status_degraded": "dégradé",
+  "octavia_load_balancer_pools_detail_overview_info_operating_status_draining": "dégradé",
+  "octavia_load_balancer_pools_detail_overview_info_operating_status_error": "erreur",
+  "octavia_load_balancer_pools_detail_overview_info_operating_status_noMonitor": "pas de Health Monitor",
+  "octavia_load_balancer_pools_detail_overview_info_operating_status_online": "online",
+  "octavia_load_balancer_pools_detail_overview_info_operating_status_offline": "offline",
+  "octavia_load_balancer_pools_detail_overview_options_title": "Options de protocole",
+  "octavia_load_balancer_pools_detail_overview_options_algorithm": "Algorithme",
```

Comment:
The values for algorithm and persistent session can be retrieved in packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/translations and don't need to be redefined here